pipelines/edx-sysadmin-publish.yml

```diff
@@ -0,0 +1,56 @@
+---
+resource_types:
+- name: pypi
+  type: docker-image
+  source:
+    repository: cfplatformeng/concourse-pypi-resource
+    tag: latest
+
+resources:
+    - name: edx-sysadmin-repo
+      type: git
+      source:
+          uri: https://github.com/mitodl/edx-sysadmin.git
+          branch: ((release))
```

Comment:
This doesn't need to be a var if we're only going to publish from that branch. You can remove the `(( ))`